server/src/main/java/module-info.java

```diff
@@ -480,4 +480,5 @@
     exports org.elasticsearch.inference.configuration;
     exports org.elasticsearch.monitor.metrics;
     exports org.elasticsearch.plugins.internal.rewriter to org.elasticsearch.inference;
+    exports org.elasticsearch.lucene.util.automaton;
```

Comment:
Including this in `server` so that we can re-use minimization in the apm module as well: https://github.com/elastic/elasticsearch/pull/119344